### PR TITLE
Bump cloudposse/tfstate-backend/aws

### PR DESCRIPTION
**Pull request by max-lobur**
## what
* Update to latest module to support AWS provider V5
* Enable SuperAdmin to assume tfstate access roles

## why
* Support AWS provider V5
* Enable SuperAdmin to use Terraform without requiring `var.privileged = true`

### CODE REVIEW

File: modules/tfstate-backend/README.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ Here's an example snippet for how to use this component.
 |------|--------|---------|
 | <a name="module_assume_role"></a> [assume\_role](#module\_assume\_role) | ../account-map/modules/team-assume-role-policy | n/a |
 | <a name="module_label"></a> [label](#module\_label) | cloudposse/label/null | 0.25.0 |
-| <a name="module_tfstate_backend"></a> [tfstate\_backend](#module\_tfstate\_backend) | cloudposse/tfstate-backend/aws | 0.38.1 |
+| <a name="module_tfstate_backend"></a> [tfstate\_backend](#module\_tfstate\_backend) | cloudposse/tfstate-backend/aws | 1.1.1 |
```
Comment (milldr) [inline modules/tfstate-backend/README.md:87]: also remove from the usage example please

File: modules/tfstate-backend/main.tf [hunk 2]
```diff
@@ -4,12 +4,13 @@ locals {
...
-  version = "0.38.1"
+  version = "1.1.1"
 
-  force_destroy                 = var.force_destroy
-  prevent_unencrypted_uploads   = var.prevent_unencrypted_uploads
-  enable_server_side_encryption = var.enable_server_side_encryption
-  enable_point_in_time_recovery = var.enable_point_in_time_recovery
+  force_destroy               = var.force_destroy
+  prevent_unencrypted_uploads = var.prevent_unencrypted_uploads
+  // enable_server_side_encryption = var.enable_server_side_encryption
```
Comment (milldr) [inline modules/tfstate-backend/main.tf:11]:
```suggestion
```

**Review by milldr — DISMISSED** [review]
nitpick, otherwise lgtm

File: modules/tfstate-backend/main.tf [hunk 3]
```diff
@@ -4,12 +4,12 @@ locals {
 
 module "tfstate_backend" {
   source  = "cloudposse/tfstate-backend/aws"
-  version = "0.38.1"
+  version = "1.1.1"
 
-  force_destroy                 = var.force_destroy
-  prevent_unencrypted_uploads   = var.prevent_unencrypted_uploads
-  enable_server_side_encryption = var.enable_server_side_encryption
```
[the remark this answers is not in the document]
Comment (max-lobur) [reply modules/tfstate-backend/main.tf:11]: Done

**Comment by max-lobur** [comment]
Not actual anymore

**Pull request closed without merging**